packages/cms-base/helpers/getTranslations.ts

```diff
@@ -0,0 +1,8 @@
+
+export default function getTranslations() {
+    try {
+        return inject("cmsTranslations", {})
+    } catch ($error) {
+        return null
```

Comment:
maybe it would be safer to return an empty object as a fallback here
```suggestion
        return {}
```

Reply:
sure, done